integration/src/test/scala/ammonite/integration/BasicTests.scala

```diff
@@ -138,6 +138,33 @@ object BasicTests extends TestSuite{
 
       assert(errorMsg.contains("IvyThing$IvyResolutionException"))
     }
+    'testIvySnapshotNoCache{
+
+      val dummyScala = intTestResources/"some-dummy-library"/'src/'main/'scala/'dummy/"Dummy.scala"
```

Comment:
Could we move this into `integration/target`? That will make sure we don't accidentally check it in since it'll be gitignored